socli/tui.py

```diff
@@ -71,10 +71,11 @@ class QuestionPage(urwid.WidgetWrap):
     def __init__(self, data):
         """
         Construct the Question Page.
-        :param data: tuple of (question_url, question_title, question_desc, question_stats, answers, comments, dup_url)
+        :param data: tuple of (question_url, question_title, question_desc, question_stats, answers, comments, dup_url, dup_link)
         """
-        question_url, question_title, question_desc, question_stats, answers, comments, dup_url = data
+        question_url, question_title, question_desc, question_stats, answers, comments, dup_url, dup_link = data
```

Comment:
Since you are unpacking the values here it may cause an error if this tuple doesn't have `dup_link` included. Please find usages of tui.QuestionPage() in the other places of the code and update the function calls there as well.